api/Window.json

```
@@ -3023,16 +3023,23 @@
             "chrome_android": {
               "version_added": "18"
             },
-            "deno": {
-              "version_added": "1.10",
-              "flags": [
-                {
-                  "type": "runtime_flag",
-                  "name": "--location",
-                  "value_to_set": "<desired origin>"
-                }
-              ]
-            },
+            "deno": [
+              {
+                "version_added": "1.16",
+                "notes": "The key used for the Web Storage bucket is based on various factors. See <a href='https://deno.land/manual/runtime/web_storage_api'>the Deno manual</a>"
```

Should I include a trailing period here or no?
```suggestion
                "notes": "The key used for the Web Storage bucket is based on various factors. See <a href='https://deno.land/manual/runtime/web_storage_api'>the Deno manual</a>."
```